src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -209,7 +210,7 @@ internal static Encoding GetOEMEncoding()
             if (s_oemEncoding == null)
             {
 #if UNIX        // PowerShell Core on Unix
-                s_oemEncoding = GetDefaultEncoding();
+                s_oemEncoding = PowerShellEncoding.GetDefaultEncoding();
 #elif CORECLR   // PowerShell Core on Windows
```

Comment:
I'll add comments about this, we can change it later after we have live-fire feedback.